_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,483 @@
+
+```html
+<html>
+  <head>
+    <title>Failed Example 1</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+  </head>
+  <body onload="registerShortcut({shortcutKey: '+', disabled: false}); activateShortcuts();">
+    <label for="target">Add to list (press "+" to add):</label>
+    <input type="text" id="target" />
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Failed Example 2
+
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false`, and there is an [instrument][] to disable the [keyboard event][] and another [instrument][] to **remap** the [keyboard event][] so that it requires that the method `getModifierState` returns `true`, but the [instruments][instrument] are not in a [clearly labeled location][].
+
+```html
+<html>
+  <head>
+    <title>Failed Example 2</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+    <script>
+      function openModal() {
+        document.getElementById('overlay').style.display = 'block'
+      }
+      function closeModal() {
+        document.getElementById('overlay').style.display = 'none'
+      }
+    </script>
+  </head>
+  <body onload="registerShortcut({id: 'singleShortcut', shortcutKey: '+'}); activateShortcuts();">
+    <div
+      style="
+        display: none;
+        position: fixed;
+        top: 2em;
+        left: 10em;
+        background-color: #505050;
+        color: white;
+        padding: 1em;
+        padding-top: 0em;
+      "
+		  id="overlay"
+	  >
+      <p>Disable/remap shortcut</p>
+      <label>
+        <input type="checkbox" onclick="toggleDisabled('singleShortcut', !this.checked)" checked>
+        Toggle single character keyboard shortcut
+      </label>
+      <br>
+      <label>
+        <input id="remap" type="checkbox" onclick="toggleModifier('singleShortcut', this.checked)">
+        Use "ctrl" key together with the "+" key
+      </label>
+      <br>
+      <button onclick="closeModal();">Dismiss</button>
+    </div>
+    
+    <input type="button" onclick="openModal()" value="Open modal" />
+    
+    <label for="target">Add to list (press "+" to add):</label>
+    <input type="text" id="target" />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+This document is not an [HTML document][].
+
+```html
+<svg height="100" width="100">
+  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
+  Sorry, your browser does not support inline SVG.  
+</svg>
+```
+
+#### Inapplicable Example 2
+
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `true`.
```

Comment:
Technically, with the current Applicability, this is Passed.
The event `keypressed "z"` is applicable. But it passes the **focus** condition because it makes no change in content (whatever has focus).

This does strongly push toward moving "make changes in content" to Applicability.
Otherwise, everything will be applicable. Event in a document with no event listener, each single key event will still be applicable and pass because it make no change in content when no `widget` is focused.

Reply:
I'm convinced. You're right, that the current applicability means everything is applicable (because it is only dependent on the event, which is controlled by the tester, without any influence from the document).